iothub/device/src/ModuleClient.cs

```diff
@@ -399,17 +401,19 @@ public void SetRetryPolicy(IRetryPolicy retryPolicy)
             InternalClient.SetMethodHandlerAsync(methodName, methodHandler, userContext, cancellationToken);
 
         /// <summary>
-        /// Registers a new delegate that is called for a method that doesn't have a delegate registered for its name.
+        /// Sets a new delegate that is called for a method that doesn't have a delegate registered for its name.
         /// If a default delegate is already registered it will replace with the new delegate.
+        /// A method handler can be unset by passing null.
```

Comment:
what do you think about elaborating this statement a bit more:
If a null delegate is passed, it will disable triggering any callback on method invocation from the service.

Reply:
I had it like that and feedback was to keep it short and simple. Is it unclear to you? If so, we should chat offline.

Reply:
I would prefer if we specified whic argument is expected to be null:
A method handler can be unset by passing a null `MethodCallback`.